core/quality/mysterium_morqa.go

```diff
@@ -288,6 +289,32 @@ func (m *MysteriumMORQA) ProviderSessions(providerID string) []ProviderSession {
 	return responseBody.Connects
 }
 
+// ProviderStatuses fetch provider connectivity statuses from quality oracle.
+func (m *MysteriumMORQA) ProviderStatuses(providerID string) (node.MonitoringAgentStatuses, error) {
+
+	request, err := m.newRequestJSON(http.MethodGet, fmt.Sprintf("providers/statuses?provider_id=%s", providerID), "")
+	if err != nil {
+		log.Error().Err(err).Msg("Failed to create provider monitoring agent statuses request")
```

Comment:
`log.Error().Err(err)` and `log.Err(err)` do the same thing, so not a lot reason to also add that `Error()`